src/desktop/screencast.rs

```diff
@@ -183,7 +183,38 @@ impl Debug for Streams {
     }
 }
 
-#[derive(Clone, Deserialize, Type)]
+/// A [builder-pattern] type to construct a response to a [`Screencast::start`]
+/// request.
+///
+/// [builder-pattern]: https://doc.rust-lang.org/1.0.0/style/ownership/builders.html
+pub struct StreamsBuilder {
+    streams: Streams,
+}
+
+impl StreamsBuilder {
+    /// Create a new instance of a streams builder.
+    pub fn new(streams: Vec<Stream>) -> Self {
+        Self {
+            streams: Streams {
+                streams,
+                restore_token: None,
+            },
+        }
+    }
+
+    /// Set the streams' optional restore token (TODO).
```

Comment:
why is this todo?

Reply:
I can't remember why I left that TODO (sigh). I'm removing it.